### DISCUSSION TITLE
How is the balancer set trigger voltage change in home assistant?

### DISCUSSION THREAD

**Discussion by oguzatagan** [syssi/esphome-jk-bms#316] [Q&A]
```
sensor:
  - platform: heltec_balancer_ble
    min_cell_voltage:
      name: "${name} min cell voltage"
      on_value_range:
        - above: 3.300
          then:
            - switch.turn_off: heltec_balancer
        - below: 3.280
          then:
            - switch.turn_on: heltec_balancer
    max_cell_voltage:
      name: "${name} max cell voltage"
      on_value_range:
        - above: 4.150
          then:
            - switch.turn_on: heltec_balancer
        - below: 4.140
          then:
            - switch.turn_off: heltec_balancer
```

**Comment by syssi** [comment]
Could you rephrase your question? I don't understand what you are looking for.

**Comment by oguzatagan** [comment]
Hello. I created triggers to balance my battery at lowest voltage and highest voltage. How can I edit my code to set and change these triggers through the home assistant interface?

**Reply by syssi** [reply]
You have to setup a template number per value.

**Comment by oguzatagan** [comment]
Hello again. I couldn't. When I write the number template in the high and low part, it underlines in red. my codes added.
```
- platform: template
    name: "Minimum Balance Trigger Voltage"
    id: minimum_balance_trigger_voltage
    optimistic: true
    min_value: 3.000
    max_value: 3.600
    step: 0.050
```

```
min_cell_voltage:
      name: "${name} min cell voltage"
      on_value_range:
        - above: minimum_balance_trigger_voltage
          then:
            - switch.turn_off: heltec_balancer
        - below: minimum_balance_trigger_voltage
          then:
            - switch.turn_on: heltec_balancer
```
